### ISSUE TITLE
Business License - Limeys Juice Bar

### ISSUE THREAD

**Issue by Zolarov** [ioShinigami/SoloRP-Directory#124]
---
name: Feature Request
about: Suggest an idea for SoloRP
title: 'Limeys Juice Bar'
labels: enhancement
assignees: 'Devs'

---

**Is your feature request related to a problem? Please describe.**
This feature is related to the lack of Juice, milkshakes and donuts in the city. I am also requesting this due to a lack of food and drinks around the city, there are very rarely places open to get more interesting items, people in the city are bored of getting burgers, sandwiches and pizzas as well as water and cola, i feel like this would give a wider variety of drinks such as Juices and Milkshakes, and foods such as donuts and crepes.

**Describe the feature you'd like**
I would like to request a business license for a Juice bar, this Juice bar is located on the corner of Vespucci Blvd between MRPD and Fleeca Bank on Legion. This would need a job much like the burger shot, the job could be called something along the lines of Juice Bar with different stages
Juice Bar Boss
- has access to a boss menu, with stash and abilities to recruit, fire and promote people and society fund
- salary of $150
Juice Bar Manager
- also has access to the boss menu, with stash and abilities to recruit, fire and promote people and society fund
- salary of $100
Juice Bar Employee
- salary of $75
I would also like a custom MLO Interior which will be linked below.
I would need many different foods and drinks that I can make, listed below 
Foods:
Donuts
Crips (chips in America)
fruit salad
chocolate bars
Drinks:
Milkshakes
- Vanilla
- Chocolate
- Strawberry
Smoothies
- Mango
- Strawberry
- Pineapple
(and the ability to mix the fruits into a mixed smoothie)
Slushies:
- Orange
- Passion Fruit
Coca Cola
Sprite/7up
Beer
Tea

Will also need interaction zones for third eye, to be able to make the smoothies milkshakes much like the burger shot. 
- First one would be on the counter and would need to option to take items from the tray
  - https://imgur.com/5zAGS9K
- Second one would need to be by the coffee machine for making coffee, milkshakes and smoothies.
   - https://imgur.com/1nMwyBV
 - Third one would need to be at the donuts stand for donuts, chips and chocolate.
    - https://imgur.com/tdE1zbX
 
**Describe alternatives you've considered**
I have considered many other different businesses such as a coffee shop however this has already been requested and a bakery. 

**Additional context**
Links to things needed:
https://www.gta5-mods.com/maps/mlo-limeys-juice-bar-fivem-add-on-sp - Limeys MLO Interior
- this has been tested on my localhost server and is fully functional

any questions please DM Zolarov#5578 on discord :)

**Comment by iBrotato** [comment]
@Zolarov, can you please reach out to a lawyer in the city, to get a proper Request for a Business License?  I'd like to attach it to this request.

**Comment by ioShinigami** [comment]
As you know Zolarov , even though you brought up the idea , you will still need the proper channels as I tell everyone else.
As for the MLO , this will need to be tested in a way other than your local. As your local does not have the same assets, database, resources or processing power as the Server itself, which is why we cant guarantee it , However I like the idea. Ill make this priority:Low , until I can accept or deny it. However as for now , it is being considered!

**Comment by Zolarov** [comment]
I appreciate it @ioShinigami, would the MLO be able to be tested in the SoloRP test server?

**Comment by ioShinigami** [comment]
Yes the test server could be a slightly valid testing environment however it still would be have to ultimately tested on the live server still. So you won't have a definite until then
